File: tests/csharp/full/code.cs
using something.baddass

/// Saying things over
/// multiple lines of code

// I love C#!!
namespace Grasses
{
  public static class Grains
  {
    public static double Square(int i) => Math.Pow(2, i - 1);

    public static double Total() => Enumerable.Range(1, 64).Select(Square).Sum();
  }
}

File: tests/php/full/code.php
<?php

namespace Lib\Exercism\Test;

use Lib\NotExercism;

// I love C#!! But get paid with PHP

/*
 * Comment
 */

public class Grains
{
    function square($n)
    {
        if ($n < 1 || $n > 64) {
            throw new InvalidArgumentException();
        }
        return pow(2, $n - 1);
    }

    function total()
    {
        return array_reduce(range(1, 64), function ($acc, $n) {
            return $acc += square($n);
        });
    }
}
